src/nbl/ext/ImGui/ImGui.cpp

```diff
@@ -483,23 +514,22 @@ namespace nbl::ext::imgui
 			IMGUI_CHECKVERSION();
 			ImGui::CreateContext();
 
-			CreateFontSampler();
-			CreateDescriptorPool();
-			CreateDescriptorSetLayout();
-			CreatePipeline(renderpass, pipelineCache);
-			CreateFontTexture(transistentCMD.get(), tQueue);
+			createFontSampler();
+			createDescriptorPool();
+			createDescriptorSetLayout();
+			createPipeline(renderpass, pipelineCache);
+			createFontTexture(transistentCMD.get(), tQueue);
 			prepareKeyMapForDesktop();
 			adjustGlobalFontScale();
-			UpdateDescriptorSets();
+			updateDescriptorSets();
 		}
 		tQueue->endCapture();
 
 		auto & io = ImGui::GetIO();
 		io.DisplaySize = ImVec2(m_window->getWidth(), m_window->getHeight());
 		io.DisplayFramebufferScale = ImVec2(1.0f, 1.0f);
 
-		m_vertexBuffers.resize(maxFramesInFlight);
-		m_indexBuffers.resize(maxFramesInFlight);
+		m_mdiBuffers.resize(maxFramesInFlight);
```

Comment:
when you'll use `StreamingTransientDataBufferST` for the mdiBuffer, you can forget about `maxFramesInFlight`

Reply:
resolved